build.rs

```diff
@@ -1,107 +1,170 @@
+// Import the `cc` crate for compiling C files
 extern crate cc;
-#[macro_use]
-extern crate cfg_if;
+use std::env;
+use std::process::Command;
+
+#[cfg(target_env = "gnu")]
```

Comment:
Is this correct? Windows is not a gnu environment...am I misunderstanding this?

Reply:
the target is namen x86_64-pc-windows-gnu 
I think this is okay